### ISSUE TITLE
Rename HumanDispersal to GravityDispersal

### ISSUE THREAD

**Issue by rafaqz** [rafaqz/Dispersal.jl#47]
It's really a gravity model - and could be useful for non-human processes with similar properties.

**Comment by rafaqz** [comment]
Or to TruncatedGravityDispersal? what do we call this thing? `SchoutenMainoGravityDispersal`? (LOL)

**Comment by rafaqz** [comment]
Any opinions on this @jamesmaino?

**Comment by jamesmaino** [comment]
True, we could rename it to Gravity. But man... who is going to be able to understand all that code!

**Comment by rafaqz** [comment]
I'm trying to make that easier lol
